index.md

```diff
@@ -68,9 +69,8 @@ fixes issue #12
 ## Introduction
 
 In software development, it's been my experience that bugs are most often introduced
-at the boundaries between applications. Unit testing works great for testing the interactions
-that an open-source maintainer knows about, but do a poor job of capturing all the
-interesting, often unexpected, ways that a community puts a library to use.
+at the boundaries between applications.
```

Comment:
I have a personal preference to linebreaks at the end of sentences in markdown [just like line 95]. It just seems like the easiest thing to maintain, say, if you wanted to add paragraphs or other content in between two sentences. It's also just a natural breaking point in writing.